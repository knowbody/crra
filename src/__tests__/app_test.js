// Generated by BUCKLESCRIPT VERSION 1.7.4, PLEASE EDIT WITH CARE
'use strict';

var App                  = require("../app.js");
var Jest                 = require("bs-jest/lib/js/src/jest.js");
var Curry                = require("bs-platform/lib/js/curry.js");
var ReactShallowRenderer = require("bs-react-test-renderer/lib/js/src/reactShallowRenderer.js");

describe("src/app.re", function () {
      Jest.test("is defined", function () {
            var component = ReactShallowRenderer.renderWithRenderer(Curry._4(App.createElement("Foo Title"), /* [] */0, /* None */0, /* None */0, /* () */0));
            return Curry._1(Jest.ExpectJs[/* toBeDefined */23], Curry._1(Jest.ExpectJs[/* expect */0], component));
          });
      return Jest.test("renders", function () {
                  var tree = ReactShallowRenderer.renderWithRenderer(Curry._4(App.createElement("Foo Title"), /* [] */0, /* None */0, /* None */0, /* () */0));
                  return Curry._1(Jest.ExpectJs[/* toMatchSnapshot */15], Curry._1(Jest.ExpectJs[/* expect */0], tree));
                });
    });

/*  Not a pure module */
