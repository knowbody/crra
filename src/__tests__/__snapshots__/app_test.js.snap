// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/app.re renders 1`] = `
<div
  className="app"
>
  <div
    className="appHeader"
  >
    <div
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "#db4d3f",
          "cursor": "pointer",
          "display": "flex",
          "width": "200px",
        }
      }
    >
      <img
        className="appLogo"
        src="test-file-stub"
      />
    </div>
    <h2
      style={
        Object {
          "fontSize": "2em",
          "marginLeft": "30px",
        }
      }
    >
      Foo Title
    </h2>
  </div>
  <p
    className="appIntro"
  >
    To get started, edit 
    <code>
      src/app.re
    </code>
     and save to reload.
  </p>
</div>
`;
